yt/yt/server/controller_agent/config.cpp

```diff
@@ -698,6 +698,10 @@ void TDockerRegistryConfig::Register(TRegistrar registrar)
 {
     registrar.Parameter("internal_registry_address", &TThis::InternalRegistryAddress)
         .Default();
+    registrar.Parameter("forward_yt_tokens_to_internal_registry", &TThis::ForwardYtTokensToInternalRegistry)
+        .Default();
+    registrar.Parameter("forward_internal_images_to_job_specs", &TThis::ForwardInternalImagesToJobSpecs)
```

Comment:
Why do we need this option?

Reply:
right now forwarding internal docker image isn't compatible with exec nodes which use porto job-env

Reply:
forwarding token without forwarding docker image makes not much sense though

Reply:
but forwarding docker image must be new default after fixing exec node